docs/examples/multi_gate.rst

```diff
@@ -93,40 +93,40 @@ logic gates:
 
 #. Single comprehensive constraint.
 
-.. testcode::
```

Comment:
Maybe better to just switch to 
1.  Single comprehensive constraint.
...
2. Multiple small constraints.
...

There aren't going to be additional sections so no need for automated counting. Sphinx renders the examples clearly as a whole with background color so flush to the side is fine, no need to tab right.

Reply:
Makes sense to me.  I switched it over to manual numbering without indentation.